server/utils/premises/occupancy.test.ts

```diff
@@ -48,4 +55,59 @@ describe('apOccupancy utils', () => {
       expect(durationSelectOptions('27')).toEqual(durationOptions)
     })
   })
+
+  describe('generateDaySummaryText', () => {
+    const buildAvailability = (overbookedCharacteristics: Array<Cas1SpaceBookingCharacteristic>) =>
+      Object.keys(occupancyCriteriaMap).map(characteristic =>
+        overbookedCharacteristics.includes(characteristic as Cas1SpaceBookingCharacteristic)
+          ? premiseCharacteristicAvailability
+              .strictlyOverbooked()
+              .build({ characteristic: characteristic as Cas1SpaceBookingCharacteristic })
+          : premiseCharacteristicAvailability
+              .available()
+              .build({ characteristic: characteristic as Cas1SpaceBookingCharacteristic }),
+      )
+
```

Comment:
I'm aware we don't have designs for overall overbooking here, but it would probably be worth adding the test and implementation now anyway, which would leave only having to update the content if required.

Reply:
Tricky. I can come up with a text for overall overbooking, but the combination of overall and characteristic overbooking might be complex.
TBH, I think there are better ways to present this information.